ArduCopter/AP_Arming.cpp

```diff
@@ -56,14 +56,19 @@ bool AP_Arming_Copter::run_pre_arm_checks(bool display_failure)
         return mandatory_checks(display_failure);
     }
 
-    return parameter_checks(display_failure)
-        & motor_checks(display_failure)
-        & pilot_throttle_checks(display_failure)
-        & oa_checks(display_failure)
-        & gcs_failsafe_check(display_failure)
-        & winch_checks(display_failure)
-        & alt_checks(display_failure)
-        & AP_Arming::pre_arm_checks(display_failure);
+    bool ret = parameter_checks(display_failure)
+                & motor_checks(display_failure)
+                & pilot_throttle_checks(display_failure)
+                & oa_checks(display_failure)
+                & gcs_failsafe_check(display_failure)
+                & winch_checks(display_failure)
+                & alt_checks(display_failure)
+                & AP_Arming::pre_arm_checks(display_failure);
+#if AP_AIRSPEED_ENABLED && !(APM_BUILD_TYPE(APM_BUILD_ArduCopter) && BOARD_FLASH_SIZE < 1025)
+    ret &= AP_Arming::airspeed_checks(display_failure);
```

Comment:
This doesn't necessarily need to be taen outside the existing structure - could just put it in the middle of the existing list...

Reply:
True. Thanks fixed!